src/fixup.js

```diff
@@ -194,7 +200,24 @@
           button.innerText = (isOpen) ? '\u25BE collapse' : '\u25B4 expand';
         }
       }
+
       document.body.appendChild(node);
+      button.focus();
+      window.onkeydown = function (event) {
+        if (event.keyCode === ESCAPEKEY && !node.classList.contains("outdated-collapsed")) {
```

Comment:
nit: you could also do here:

```
var isCollapsed = node.classList.contains("outdated-collapsed");
```

In case you need to add another thing to the if later.

Reply:
done